keanu-project/src/main/java/io/improbable/keanu/kotlin/DoubleOperators.kt

```diff
@@ -1,17 +1,62 @@
 package io.improbable.keanu.kotlin
 
 
-interface DoubleOperators<T> : Operators<T> {
+interface DoubleOperators<T> : NumberOperators<T> {
 
-    operator fun minus(that: Double): T
     operator fun plus(that: Double): T
+
+    operator fun minus(that: Double): T
+
+    /**
+     * @return that - this
+     */
+    fun doubleMinusThis(that: Double): T
```

Comment:
This is a very cool feature. Doesn't get used often but it's pretty key when it is. It's called reverse minus in nd4j and may have another name numpy. Can you find a reasonable name and use that?

Reply:
Yes! Will try and generalise it to division as well

Reply:
Looks like it's `rsub` in numpy too. I'll go with `reverseMinus` and `reverseDiv`